pkg/xds/envoy/listeners/v3/dns_configurer.go

```diff
@@ -52,32 +47,8 @@ func (c *DNSConfigurer) dnsFilter() *envoy_dns.DnsFilterConfig {
 	}
 	sort.Stable(DnsTableByName(virtualDomains)) // for stable Envoy config
 
-	resolverConfig := envoy_cares.CaresDnsResolverConfig{
-		Resolvers: []*envoy_core.Address{
-			{
-				Address: &envoy_core.Address_SocketAddress{
-					SocketAddress: &envoy_core.SocketAddress{
-						Address: "127.0.0.1",
-						PortSpecifier: &envoy_core.SocketAddress_PortValue{
-							PortValue: c.EmptyDNSPort,
-						},
-					},
-				},
-			},
-		},
-	}
-
 	return &envoy_dns.DnsFilterConfig{
 		StatPrefix: "kuma_dns",
-		ClientConfig: &envoy_dns.DnsFilterConfig_ClientContextConfig{
-			// We configure upstream resolver to resolver that always returns that it could not find the domain (NXDOMAIN)
-			// As for this moment there is no setting to disable upstream resolving.
-			TypedDnsResolverConfig: &envoy_core.TypedExtensionConfig{
-				Name:        "envoy.network.dns_resolver.cares",
-				TypedConfig: util_proto.MustMarshalAny(&resolverConfig),
-			},
-			MaxPendingLookups: 256,
```

Comment:
I guess we're intentionally taking the default value?

Reply:
yes. Default value is "do not pass this to external DNS"